index.html

```diff
@@ -2077,6 +2077,20 @@
 configure their agents to respond with other information, for reasons including testing, auditing or
 mitigating forms of data collection, including <a>browser fingerprinting</a>.
 
+## Centralization
+
+Specification authors, browser vendors, and other relevant parties should be careful to avoid designing or
+implementing systems that would increase centralization when implementing the privacy principals listed in
```

Comment:
```suggestion
implementing systems that would increase centralization when implementing the privacy principles listed in
```

Reply:
Also: I think it would be good to include a reference to https://datatracker.ietf.org/doc/html/draft-nottingham-avoiding-internet-centralization.